### DISCUSSION TITLE
billing_code_modifier and service_code arrays containing single element of "null"

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#573] [General]
The schema description for in_network files seems to require the service_code array to be "An array of two-digit strings", but we are seeing "null" in the array in a 08-01 UHC file.  

This is also occurring for the billing_code_modifier.  The definition in the schema for billing_code_modifier is not as restrictive, but one would hope that the array elements are strings containing valid code modifiers and "null" isn't one.  Is there any reason for this to be a "null"?

Anyone else see these?

This seems like a new bug since the 07 files didn't contain any modifiers.

Here is an example:

```
{
  "negotiation_arrangement": "ffs",
  "name": "COLONOSCOPY FLX DX W/COLLJ SPEC WHEN PFRMD",
  "billing_code_type": "CPT",
  "billing_code_type_version": "2022",
  "billing_code": "45378",
  "description": "Colonoscopy, flexible; diagnostic, including collection of specimen(s) by brushing or washing, when performed (separate procedure)",
  "negotiated_rates": [
    {
      "provider_references": [
        30
      ],
      "negotiated_prices": [
        {
          "negotiated_rate": 175.53,
          "service_code": [
            "null"
          ],
          "negotiated_type": "negotiated",
          "expiration_date": "9999-12-31",
          "billing_class": "professional",
          "billing_code_modifier": [
            "null"
          ],
          "additional_information": ""
        },
        {
          "negotiated_rate": 317.34,
          "service_code": [
            "null"
          ],
          "negotiated_type": "negotiated",
          "expiration_date": "9999-12-31",
          "billing_class": "professional",
          "billing_code_modifier": [
            "null"
          ],
          "additional_information": ""
        }
      ]
    },
```

I am happy to share more details, if that is necessary to track down the issue.

**Comment by bryanbriegel** [comment]
I dont have an answer on this one, however it would be fantastic if the schema could be updated from "service_code" to "place_of_service_code" as it is specified in the PRA packet, and since it is in fact place of service and would disambiguate from "billing_code"

**Reply by nmayle** [reply]
It will be great when we get to the point where the biggest issue is the cases where the field names are confusing.

**Reply by bryanbriegel** [reply]
usually, its a great starting point -  8)
